src/domains/home/HomeScreen.tsx

```diff
@@ -57,13 +58,14 @@ const Section: FC<{ title: string }> = ({ children, title }) => {
 
 export const HomeScreen: FC = () => {
   const isDarkMode = useColorScheme() === 'dark';
+  const navigation = useNavigation<RootStackNavigationProp<'BottomTabNavigator'>>();
```

Comment:
Why is `BottomTabNavigator` passed in here?

Reply:
Fixed this.